mobile/src/components/uam-configure-group/helpers/get-selected-items-values/get-selected-items-values.helper.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  PermissionsGetAllItemResponseDto,
+  UsersGetResponseDto,
+} from '~/common/types/types';
+import { SelectedItemsValues } from '~/components/uam-configure-group/common/types/types';
+
+type Props = {
+  checkedIds: number[];
+  items: (UsersGetResponseDto | PermissionsGetAllItemResponseDto)[];
+  namePrefix: string;
+};
+
+const getSelectedItemsValues = ({
+  checkedIds,
+  items,
+  namePrefix,
+}: Props): SelectedItemsValues => {
```

Comment:
Remove SelectedItemsValues type
```suggestion
}: Props): Record<string, boolean> => {
```